timemachine/cpp/src/kernels/k_nonbonded.cu

```diff
@@ -9,6 +9,7 @@ void __global__ k_coords_to_kv(
     unsigned int *vals) {
 
     const int atom_idx = blockIdx.x * blockDim.x + threadIdx.x;
+    const int N_BINS = 128;
```

Comment:
nit: Would be good to have a single definition of this variable rather than three. That said, not obvious where that should be.

Reply:
Agreed! To define this in a single place, cleanest approach I could think of was to expose N_BINS as a parameter in a few places, and then set the default value for that parameter once. However, I initially wanted to avoid changing any signatures.

Reply:
Did you consider putting it in `k_nonbonded.cuh`? (I don't immediately see an issue with this)

Reply:
> nit: `N_BINS` seems slightly misleading to me, since it seems like there are really `N_BINS * N_BINS` "bins". Would something like `hilbert_grid_size` be any clearer?

Agreed -- it's really `N_BINS**3`... Grid size or similar would be an improvement! Will fix...

Reply:
Thanks for these suggestions! Consolidated the definitions into `k_nonbonded.cuh` as suggested: https://github.com/proteneer/timemachine/pull/692/commits/388e71ed81be993239bf2581ebc4d22067233776 . Also renamed from misleading "N_BINS" to "GRID_SIZE"